src/Screens/Main/Chat/MessageList/Message.tsx

```diff
@@ -16,28 +11,27 @@ export type MessageProps = {
   type: 'Message';
   value: messageApi.Message;
   id: string;
+  isSeen: boolean;
 };
 
 const Message = ({ value: message }: MessageProps) => {
   const { content, sentTime, type } = message;
-  const dispatch = useDispatch<redux.Dispatch<actions.Action>>();
-  React.useEffect(() => {
-    if (!message.isSeen && message.type === 'Received') {
-      dispatch(markSeen({ message }));
-    }
-  }, []);
 
   const bubbleStyle =
     type === 'Received' ? styles.leftBubble : styles.rightBubble;
 
+  // TODO: Visual aid, remove this after review
+  const isSeenBorder = type === 'Received' &&
+    !message.isSeen && { borderWidth: 2, borderColor: colors.orangeLight };
+
```

Comment:
Remove this, will be implemented in https://trello.com/c/8XmKPIUO/944-design-unread-message-visual-aid